docs/faq.md

```diff
@@ -0,0 +1,53 @@
+Using the recommended setup, you setup your import map to download that once. Then, tell each application to _not_ bundle that application code; instead, it will given to you at runtime in the browser. See webpack’s “externals” docs for how to do this.
+
+You do have the option of _not_ excluding those libraries (for example if you want to experiment with a newer version of a different library alltogether) but be aware of the effect that will have on user's bundle sizes and application speed.
+
+## What are import maps?
+[Import maps](https://github.com/WICG/import-maps) improve the developer experience of ES modules by allowing you to write something like `import React from "react"` instead of needing to use an absolute or relative URL for your import statement. The same is also true of importing from other single-spa applications, e.g. `import {MyButton} from "styleguide"`. The import-map spec is currently in the process of being accepted as a web standard and at the time of writing has been implemented in Chrome.
+
+## How can I share application state between two or more applications?
+In general, we recommend trying to avoid this — it couples those apps together. If you find yourself doing this frequently between apps, you may want to consider that those separate apps should actually just be one app.
+
+Generally, it’s better to just make an API request for the data that each app needs, even if parts of it have been requested by other apps. In practice, if you’ve designed your application boundaries correctly, there will end up being very little application state that is truly shared — your friends list has different data requirements than your social feed.
+
+However, that doesn’t mean it can’t be done. Here are several ways:
+1. Create a shared API request library that can cache requests and their responses. If somone hits an API, and then that API is hit again by another application, it just uses the cache
+2. Expose the shared state as an export, and other libraries can import it. Observables (like [RxJS](https://rxjs-dev.firebaseapp.com/)) are useful here since they can stream new values to subscribers
+3. Use [custom browser events](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events#Creating_custom_events) to communicate
+
+## Should I use microservices on the frontend?
+If you’ve ran into some of the headaches a monolithic repo has, then you should really consider it.
+
+In addition, if your organization is setup in a Spotify-type model (e.g. where there are autonomous squads that own full-stack features) then microservices on the frontend will fit very well into your setup.
+
+However, if you’re just starting off and have a small project or a small team, we would recommend you stick with a monolith (i.e. not microservices) until you get to the point that scaling is getting hard. (Don’t worry, we’ll be here to help you migrate)
```

Comment:
maybe clarify that "scaling" refers to "scaling your organization for many features and developers". A lot of people might misread as amount of data, amount of customers, etc.